docs/kratos/social-signin/45_slack.mdx

```diff
@@ -6,7 +6,61 @@ sidebar_label: Slack
 
 # Slack
 
-Follow these steps to add Spotify as a social sign-in provider to your project using the Ory CLI:
+````mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add Slack as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the Slack logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a Slack OAuth Application](https://api.slack.com/apps) by clicking the **Create New App** button
```

Comment:
For me the button is labeled differently: 

![image](https://user-images.githubusercontent.com/29116492/214830684-85774d9b-2761-4774-b3d7-66fa60319e47.png)


Better to stick with "create a slack oauth app" with link IMO.

Reply:
Here's what I see when I'm logged into the developer site:

![D9CA8D16-0BB6-4643-BDDE-B269E20190E5](https://user-images.githubusercontent.com/397343/214835641-86354d61-0ea8-4070-adb6-89dbb551c97d.jpeg)

Reply:
When you don't have any apps created you get the view and button I got. When you have one or more apps created, you get your variant. I'm removing the part of the sentence with the button name.